lighthouse-core/test/test-env/mocha-setup.js

```diff
@@ -0,0 +1,210 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
```

Comment:
File was written in 2021, in general I'm not keen on updating the year esp. since there's a general sense these don't particularly matter... we just do it because we always have...